src/core/components/Paginate/index.js

```diff
@@ -29,7 +29,7 @@ export class PaginateBase extends React.Component {
 
   static defaultProps = {
     LinkComponent: Link,
-    perPage: 20,
+    perPage: 25,
```

Comment:
Can you add a comment saying that this is hard coded to the API's default page size?